cli/support.go

```diff
@@ -0,0 +1,209 @@
+package cli
+
+import (
+	"archive/zip"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"golang.org/x/xerrors"
+
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/sloghuman"
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/support"
+)
+
+func (r *RootCmd) support() *clibase.Cmd {
+	supportCmd := &clibase.Cmd{
+		Use:   "support { dump }",
```

Comment:
Just spotted, what's the purpose of `{ dump }` here?

Reply:
Ah I forgot that `clibase` will handle showing the subcommands.